core/state/node/core.go

```diff
@@ -52,3 +52,9 @@ type ReverseProxyRule struct {
 	Target  string `json:"target"`
 	AppID   string `json:"app_id"`
 }
+
+const (
+	ProxyRuleFileServer       = "file"
+	ProxyRuleRedirect         = "redirect"
```

Comment:
also can we move all of these into the `reverse_proxy` package?

so we can do stuff like `reverse_proxy.Rule` instead of `node.ReverseProxyRule`

Reply:
I don't thinks so without creating a bunch of import loops. These were originally in there, but they moved once the reverse proxy rules became a part of the state.